runtime/port/unix/j9sysinfo.c

```diff
@@ -622,6 +626,10 @@ getAIXPPCDescription(struct J9PortLibrary *portLibrary, J9ProcessorDesc *desc)
 		desc->physicalProcessor = PROCESSOR_PPC_P7;
 	} else if (__phy_proc_imp_8()) {
 		desc->physicalProcessor = PROCESSOR_PPC_P8;
+#if defined(J9OS_I5)
```

Comment:
Does `J9OS_I5_V6R1` imply `J9OS_I5`?  Or are the two ifdefs completely distinct?

Reply:
J9OS_I5 can cover all os releases on IBM i and J9OS_I5_V6R1 is only for one release.

Reply:
So `J9OS_I5_V6R1` will never run on a `PROCESSOR_PPC_P9`?  I'm surprised by that restriction but if that's the intention then this fine.

Reply:
yes, J9OS_I5_V6R1 is an old release.